src/views/v2/widgets/events-list.php

```diff
@@ -73,26 +73,34 @@
 		data-view-breakpoint-pointer="<?php echo esc_attr( $breakpoint_pointer ); ?>"
 	<?php endif; ?>
 >
-	<div class="tribe-events-widget-events-list">
-		<header class="tribe-events-widget-events-list__header">
-			<h3 class="tribe-events-widget-events-list__header-title tribe-common-h6 tribe-common-h--alt">
-				<?php
-				echo esc_html(
-					sprintf(
-						_x( 'Upcoming %1$s', 'Title for events list widget.', 'the-events-calendar' ),
-						tribe_get_event_label_singular()
-					)
-				);
-				?>
-			</h3>
-		</header>
+	<div class="tribe-common-l-container">
+		<div class="tribe-events-widget-events-list">
+			<header class="tribe-events-widget-events-list__header">
+				<h3 class="tribe-events-widget-events-list__header-title tribe-common-h6 tribe-common-h--alt">
+					<?php
+					echo esc_html(
+						sprintf(
+							_x( 'Upcoming %1$s', 'Title for events list widget.', 'the-events-calendar' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 54540900282e5c3a5e155813f20f4e88</sub>